tools/create_index/index_generator.js

```diff
@@ -14,7 +14,7 @@ const supportedVersions = ['14.0.0', '16.0.0', '18.0.0', '19.0.0']
 
 const supportedVersionAffected = function (json) {
   for (const version of supportedVersions) {
-    if (satisfies(version, json.vulnerable)) {
+    if (!json.vulnerable || satisfies(version, json.vulnerable)) {
```

Comment:
You can move it to before the `for` loop. Including a comment would be great too.

Reply:
I made a refactor in https://github.com/nodejs/security-wg/pull/920/commits/c7cfd6261ecd1a7174e42c8185aefd893c0d9af8. Thanks for the suggestion, @RafaelGSS :)